web/rainmaker/dev-packages/egov-workflow-dev/src/ui-containers-local/WorkFlowContainer/index.js

```diff
@@ -251,10 +232,7 @@ class WorkFlowContainer extends React.Component {
           label
         )}&applicationNumber=${applicationNumber}&tenantId=${tenant}&secondNumber=${licenseNumber}&moduleName=${moduleName}`;
 
-        if (moduleName === "NewWS1" || moduleName === "NewSW1") {
-          window.location.href = `acknowledgement?${this.getPurposeString(label)}&applicationNumber=${applicationNumber}&tenantId=${tenant}`;
-        }
-
+        window.location.href = `acknowledgement?${this.getPurposeString(label)}&${redirectQueryString}`;
```

Comment:
what is  redirectquery string ? what if no redirectquery is passed , as it is a common component ,it might affect other modules

Reply:
We are passing redirectQueryString as props from search-preview page becasue based on apply it will go to one url.if it is modify it will take to another relevant URL. code is written inside WNS module.it will not effect other module.